app/helpers/application_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ApplicationHelper
+  class << self
```

Comment:
Might be better to place this logic inside of some other helper rather than the most generic/base one.